components/browser/storage-sync/pings.yaml

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/pings/1-0-0
+
+history_sync:
+  description: >
```

Comment:
heads-up: please note that once [this](https://github.com/mozilla/glean_parser/pull/70) is merged, a `data_reviews` field will be required for each ping as well. You can be future proof by adding this already, so you won't need to change it later.